Panic error while upgrading from 1.4.1 to 1.5

<!-- Please reserve GitHub issues for bug reports and feature requests.

For questions, the best place to get answers is on our [discussion forum](https://discuss.hashicorp.com/c/vault), as they will get more visibility from experienced users than the issue tracker.

Please note: We take Vault's security and our users' trust very seriously. If you believe you have found a security issue in Vault, please responsibly disclose by contacting us at security@hashicorp.com. Our PGP key is available at [our security page](https://www.hashicorp.com/security/).

-->

**Describe the bug**
I am upgrading vault cluster using helm charts.


Error message:
==> Vault server configuration:

 OCI KMS Crypto Endpoint: https://xxxxxx-xx.xxx.xxx-xxxx-1.xxxxxxcloud.com
           OCI KMS KeyID: ocid1.key.oc1.xxx.xxxxxxxx.xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
panic: strings: negative Repeat count

goroutine 1 [running]:
strings.Repeat(0x37ac058, 0x1, 0xfffffffffffffffd, 0x3, 0x3)
        /usr/local/go/src/strings/strings.go:533 +0x5a9
github.com/hashicorp/vault/command.(*ServerCommand).Run(0xc0000cf340, 0xc00004c0b0, 0x1, 0x1, 0x0)
        /go/src/github.com/hashicorp/vault/command/server.go:1525 +0x3a6a
github.com/hashicorp/vault/vendor/github.com/mitchellh/cli.(*CLI).Run(0xc000274140, 0xc000274140, 0xc00038a520, 0xc00038a420)
        /go/src/github.com/hashicorp/vault/vendor/github.com/mitchellh/cli/cli.go:255 +0x1da
github.com/hashicorp/vault/command.RunCustom(0xc00004c0a0, 0x2, 0x2, 0xc000136190, 0xc000096058)
        /go/src/github.com/hashicorp/vault/command/main.go:180 +0x834
github.com/hashicorp/vault/command.Run(...)
        /go/src/github.com/hashicorp/vault/command/main.go:88
main.main()
        /go/src/github.com/hashicorp/vault/main.go:10 +0x71

**To Reproduce**
Steps to reproduce the behavior:
1. Run `vault write ...`
2. Run `vault login....`
3. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Environment:**
current vault version: 1.4.1
target vault version: 1.5.0
Deployed in Kubernetes cluster using helm charts

Vault server configuration file(s):

ui = true
        plugin_directory = "/vault/data/plugins/"
        log_level = "trace"
        disable_mlock = true
        api_addr = "[::]:8200"
        listener "tcp" {
          tls_disable = 0
          address = "[::]:8200"
          cluster_address = "[::]:8201"
          tls_min_version = "tls12"
          tls_cert_file = "/vault/userconfig/vault-server-tls/vault.crt"
          tls_key_file  = "/vault/userconfig/vault-server-tls/vault.key"
          tls_client_ca_file = "/vault/userconfig/vault-server-tls/vault.ca"
        }

        storage "raft" {
          path = "/vault/data/"
        }
        telemetry {
          prometheus_retention_time = "30s",
          disable_hostname = true
        }
        seal "ocikms" {
          key_id               = "ocid1.key.oc1.xxx.xxxxxxxx.xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
          crypto_endpoint      = "https://xxxxxxxxx-xx.xxx.xx-xxxx-1.xxxxxcloud.com"
          management_endpoint  = "https://xxxxxxxxx-xxxxxxxxx.kms.xx-xxxxx-1.oxxxcloud.com"
          auth_type_api_key    = "false"
        }

        service_registration "kubernetes" {}
```

--- a/command/server.go
+++ b/command/server.go
@@ -1513,16 +1513,12 @@ CLUSTER_SYNTHESIS_COMPLETE:
 	infoKeys = append(infoKeys, "go version")
 	info["go version"] = runtime.Version()
 
-	// Server configuration output
-	padding := 24
-
 	sort.Strings(infoKeys)
 	c.UI.Output("==> Vault server configuration:\n")
 
 	for _, k := range infoKeys {
 		c.UI.Output(fmt.Sprintf(
-			"%s%s: %s",
-			strings.Repeat(" ", padding-len(k)),
+			"%24s: %s",
 			strings.Title(k),
 			info[k]))
 	}

